README.md

```diff
@@ -232,7 +232,7 @@ Other Style Guides
   <a name="es6-object-concise"></a>
   - [3.6](#3.6) <a name='3.6'></a> Use property value shorthand.
 
-  > Why? It is shorter to write and descriptive.
+  > Why? It is shorter to write and autological.
```

Comment:
Autological = describes itself. Could use self-descriptive instead, I just don't see 'descriptive' as the right word here.